src/orbit-db-identity-provider.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+const IdentityProvider = require('./identity-provider')
+const Keystore = require('orbit-db-keystore')
+const keypath = './orbitdb/identity/keys'
+const type = 'orbitdb'
+
+class OrbitDBIdentityProvider extends IdentityProvider {
+  constructor (options = {}) {
+    super()
+    this._keystore = Keystore.create(options.keypath || keypath)
```

Comment:
This whole file is looking very good! 👍 🎉 Let's do the `keypath` renaming and this should be ready.